test/NomoPlayersDropMechanic.test.ts

```diff
@@ -479,7 +535,11 @@ describe("NomoPlayersDropMechanic tests", function () {
         deployerAddress,
         tokenPrice,
         maxQuantity,
-        testCollectibleItems) as NomoPlayersDropMechanic;
+        vrfCoordinatorAddress,
```

Comment:
instead of deploying new instance can we just force failure by buying tokens so that we ensure less tokens than the `privileged` accounts are in order to fulfil the test